tools/accuracy_checker/accuracy_checker/metrics/text_detection.py

```diff
@@ -150,7 +151,13 @@ def configure(self):
 
     def update(self, annotation, prediction):
         gt_rects = list(map(rect_from_points, annotation.boxes))
-        prediction_rects = list(map(rect_from_points, prediction.boxes))
+        if 'TextDetectionPrediction' in str(type(prediction)):
```

Comment:
please use `isinstance` for type checking

Also I have alternative suggestion, you can just add boxes property in DetectionPrediction type for align behaviour between 2 types and hide boxes calculation implementation. For me, it can be more useful, because if we will need to extend support with another prediction type or introduce new metric, we will not care how to find boxes

Reply:
Thank you for your quick and detailed response. I've fixed all your comments